scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -946,6 +1037,11 @@ Function PrintHighlightedMessage
 }
 
 $Architecture = GetArchitecture
+$OptimizeForPerformance=$True
+If ($Architecture -eq "arm32v7") 
+{
```

Comment:
Do we still need this?

Reply:
Yes, since iotedgequickstart default value is true; therefore we need to explicitly set to false for arm32v7.